src/rofi_rbw/selector.py

```diff
@@ -84,7 +84,12 @@ def _format_folder(entry: Entry, show_folders: bool) -> str:
 
     @staticmethod
     def justify(entry: Entry, max_width: int, show_folders: bool) -> str:
-        return " " * (max_width - len(entry.name) - ((len(entry.folder) + 1) if show_folders else 0))
+        n = max_width - len(entry.name)
```

Comment:
Could you please rename `n` to something more descriptive, for example `whitespace_length`?